official/resnet/imagenet_main.py

```diff
@@ -102,10 +102,9 @@ def parse_record(raw_record, is_training):
   # Note that the resulting image contains an unknown height and width
   # that is set dynamically by decode_jpeg. In other words, the height
   # and width of image is unknown at compile-time.
-  # Results in a 3-D int8 Tensor which we then convert to a float
-  # with values ranging from [0, 1).
+  # Results in a 3-D int8 Tensor. This will be converted to a float later,
+  # during resizing.
   image = tf.image.decode_jpeg(image, channels=_NUM_CHANNELS)
```

Comment:
Are the numbers scaled to between [0,1] later on?

Reply:
As far as I can tell, no, though it doesn't affect convergence in our tests.

Reply:
Hmm, that's good to hear. I would still consider dividing the image values by 255, because during preprocessing the means subtracted from the image are scaled to [0,1].

Reply:
Interesting. That implies that maybe it is scaled later on? Let me go through again and double-check.

Reply:
It looks like the means were still being scaled, but not the final values-- implying mean subtraction doesn't matter a ton? Is that plausible? I removed the scaling from the means, and am running now to validate convergence.